cico_run_tests.sh

```diff
@@ -1,6 +1,40 @@
 #!/bin/bash
+# Output command before executing
```

Comment:
I understand your intention here to have a script to be able to trigger the test job. But this means we again have our build/deployment code in keycloak master which will make rebasing from upstream not trivial: the same issue we are trying to solve by splitting the repo.